### DISCUSSION TITLE
Remove link to file in document chooser

### DISCUSSION THREAD

**Discussion by enzedonline** [wagtail/wagtail#8621] [Ideas]
When choosing a document, the file name is linked to the file itself instead of the chooser url. Clicking on it downloads the file rather than selects the document.

![image](https://user-images.githubusercontent.com/72310199/171616365-b18aea42-4675-4cae-bf10-c7fc4c5e9ba2.png)
![image](https://user-images.githubusercontent.com/72310199/171616426-9bef2b19-88e0-4ac3-9213-10ebad1b6cfe.png)

It's a bit confusing for the editor - not really the point where anybody is going to want to download the document.

Would be better to either not have a link there or to link to the document chooser (i.e. http://example.com/admin/documents/chooser/n/).

**Comment by gasman** [comment]
I can't say for sure how much that link gets used, but I can imagine potential uses for it (namely, if several documents have ended up with similar names and the editor needs to view it to make sure they're linking to the right one), so I'd be reluctant to remove it entirely in case someone's relying on it.

Maybe the UI could be improved though - perhaps the link could be alongside the filename with the text "[download]", or maybe there's a good icon we can use to communicate this.
